src/locale/kz/translations.json

```diff
@@ -421,6 +421,9 @@
       "no": "Жоқ",
       "restore": "Қалпына келтіру",
       "close": "жабық"
+    },
+    "accommodation_bookings": {
```

Comment:
why is it different, you wrote in translation english
```
"accommodation_reservations": {
      "all_accommodation_reservations": "All reservations"
    }
```
now you used accommodation_bookings, change accommodation_bookings =>accommodation_reservations